install/ci-vm/ci-linux/ci/runCI

```diff
@@ -61,13 +61,14 @@ echo "" > "${logFile}"
 
 postStatus "building" "Checking if CCExtractor build artifact is present"
 if [ -e "${dstDir}/ccextractor" ]; then
-        chmod 700 "${dstDir}/ccextractor"
+        cp ${dstDir}/ccextractor ./
+        chmod 700 ccextractor
```

Comment:
Just curious, why do we need to move the file outside and then alter the permissions?

Reply:
We can do either (i.e. change permissions and then copy), it is just that we cannot run that file within that folder, because we got a Permission issue, when we did so.